codewars4.py

```diff
@@ -0,0 +1,25 @@
+# https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/python
+
+# def get_count(sentence):
+#     pass
+
+vowels_count = 0
+
+sentence = input("Please write something: ")
+
+for char in sentence:
+    if char.casefold() == "a" or char.casefold() == "e" or char.casefold() == "i" or char.casefold() == "o" or char.casefold() == "u":
+        vowels_count += 1
+
+print(vowels_count)
+
+# code to enter into codewars terminal:
+#
+# def get_count(sentence):
+#     vowels_count = 0
+#
+#     for char in sentence:
+#          if char.casefold() == "a" or char.casefold() == "e" or char.casefold() == "i" or char.casefold() == "o" or char.casefold() == "u":
+#             vowels_count += 1
+#
+#     return (vowels_count)
```

Comment:
If you want to avoid doing two versions of code, you can just use it and test it with the CodeWars version, where you use the function. The whole body of a function (lines 19-25 in your exercise) is where you declare your function. It doesn't do anything, until you _call the function_. To do so, simply use its name and pass a variable, so:
1. Leave line 26 empty
2. In line 27 call function, by it's name: _get_count()_ - just don't forget a word in parenthesis. This word will be processed by the function.

Reply:
Whatever makes you feel more comfortable right now is best :) Great job overall, really!